test/integration/good/code-gen/cpp.expected

```diff
@@ -4269,7 +4243,7 @@ f12(const int& a1, const std::vector<int>& a2,
   local_scalar_t__ DUMMY_VAR__(std::numeric_limits<double>::quiet_NaN());
   (void) DUMMY_VAR__;  // suppress unused var warning
   try {
-    current_statement__ = 760;
+    current_statement__ = 749;
```

Comment:
Hmm, I need to look into why it's changing these current statements. Seems weird?